<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrmContractEdit</class>
 <widget class="QDialog" name="FrmContractEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>683</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3" rowstretch="1,9,0">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="labTitle">
        <property name="text">
         <string>合同管理</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QWidget" name="widget_3" native="true">
        <layout class="QGridLayout" name="gridLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="QToolButton" name="toolButton">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QVBoxLayout" name="verticalLayout" stretch="9,1">
      <item>
       <widget class="QWidget" name="widget_4" native="true">
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>开始日期：</string>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>结束日期：</string>
           </property>
          </widget>
         </item>
         <item row="0" column="7">
          <widget class="QLabel" name="label_15">
           <property name="text">
            <string>粮食类型:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="8">
          <widget class="QComboBox" name="cbxGrainType">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>0</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="0" column="9">
          <spacer name="horizontalSpacer_6">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>113</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>合同类型:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QComboBox" name="cbxContractType">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>0</height>
            </size>
           </property>
           <property name="duplicatesEnabled">
            <bool>false</bool>
           </property>
           <property name="frame">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>105</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="4">
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>采购总量:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="5">
          <widget class="QDoubleSpinBox" name="dbSpinBoxTotal">
           <property name="suffix">
            <string>吨</string>
           </property>
          </widget>
         </item>
         <item row="1" column="6">
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>106</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="7">
          <widget class="QLabel" name="label_17">
           <property name="text">
            <string>已完成:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="8">
          <widget class="QDoubleSpinBox" name="dbSpinBoxConmpleted">
           <property name="suffix">
            <string>吨</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>合同号码:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2" colspan="2">
          <widget class="QLineEdit" name="editContractNumber">
           <property name="inputMask">
            <string/>
           </property>
          </widget>
         </item>
         <item row="2" column="4">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>订单编号:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="5" colspan="2">
          <widget class="QLineEdit" name="editOrder"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>原发地:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="2" colspan="2">
          <widget class="QLineEdit" name="editSource"/>
         </item>
         <item row="3" column="4">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>供应商:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="5">
          <widget class="QComboBox" name="cbxSupplier">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>0</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="3" column="7">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>运输公司:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="8">
          <widget class="QComboBox" name="cbxTransport"/>
         </item>
         <item row="3" column="9">
          <spacer name="horizontalSpacer_7">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>105</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_14">
           <property name="text">
            <string>车皮号码:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="2" colspan="2">
          <widget class="QLineEdit" name="editWagonID"/>
         </item>
         <item row="4" column="4">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>车船号码:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="5" colspan="2">
          <widget class="QLineEdit" name="editTransportID"/>
         </item>
         <item row="5" column="0" colspan="2">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>银行类型:</string>
           </property>
          </widget>
         </item>
         <item row="5" column="2">
          <widget class="QComboBox" name="cbxBankType"/>
         </item>
         <item row="5" column="3">
          <spacer name="horizontalSpacer_8">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>105</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="5" column="4">
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>支付方式:</string>
           </property>
          </widget>
         </item>
         <item row="5" column="5">
          <widget class="QComboBox" name="cbxPayType"/>
         </item>
         <item row="5" column="6">
          <spacer name="horizontalSpacer_9">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>106</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="5" column="7">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>银行卡号:</string>
           </property>
          </widget>
         </item>
         <item row="5" column="8" colspan="2">
          <widget class="QLineEdit" name="editBankID">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="label_16">
           <property name="text">
            <string>计价方式:</string>
           </property>
          </widget>
         </item>
         <item row="6" column="2">
          <widget class="QComboBox" name="cbxValuation"/>
         </item>
         <item row="6" column="3">
          <spacer name="horizontalSpacer_10">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>105</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="6" column="5">
          <widget class="QCheckBox" name="cbxIsComplete">
           <property name="text">
            <string>是否完成</string>
           </property>
          </widget>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="label_18">
           <property name="text">
            <string>联系人:</string>
           </property>
          </widget>
         </item>
         <item row="7" column="4">
          <widget class="QLabel" name="label_19">
           <property name="text">
            <string>联系电话:</string>
           </property>
          </widget>
         </item>
         <item row="7" column="5" colspan="2">
          <widget class="QLineEdit" name="editPhone"/>
         </item>
         <item row="8" column="0">
          <widget class="QLabel" name="label_20">
           <property name="text">
            <string>备注:</string>
           </property>
          </widget>
         </item>
         <item row="7" column="2" colspan="2">
          <widget class="QLineEdit" name="editContacts"/>
         </item>
         <item row="8" column="2" colspan="5">
          <widget class="QLineEdit" name="editMark"/>
         </item>
         <item row="0" column="2">
          <widget class="QDateEdit" name="dateEditStart">
           <property name="calendarPopup">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="5">
          <widget class="QDateEdit" name="dateEditEnd">
           <property name="calendarPopup">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="topMargin">
         <number>60</number>
        </property>
        <property name="bottomMargin">
         <number>20</number>
        </property>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btnEdit">
          <property name="text">
           <string>修改</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnEdit_2">
          <property name="text">
           <string>返回</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
